README.md

```diff
@@ -437,7 +439,33 @@ $ docker run -d -it -p 7200:8000 -p 7201:8001 -p 7202:8002 \
      store/marklogicdb/marklogic-server:10.0-9-centos-1.0.0-ea4
 ```
 
-When you complete these steps, you will have multiple containers; one on each VM and all connected to each other on the 'ml-cluster-network' network. All the containers will be part of same cluster.
+When you complete these steps, you will have multiple containers; one on each VM, and all connected to each other on the 'ml-cluster-network' network. All the containers will be part of same cluster.
+
+# Upgrading to the latest MarkLogic Docker Release
+
+MarkLogic has extensive documentation about upgrades, see [https://docs.marklogic.com/guide/relnotes/chap2](https://docs.marklogic.com/guide/relnotes/chap2). Other than the uninstall and install of the MarkLogic RPMs, the overall processes and compatibility notes for upgrades remain the same when you run MarkLogic in containers. Instead of uninstalling and installing the MarkLogic RPMs, use the following procedure to upgrade a container instance to a newer release of MarkLogic. Be sure to follow the sequence described in the documentation for rolling upgrades [https://docs.marklogic.com/guide/admin/rolling-upgrades](https://docs.marklogic.com/guide/admin/rolling-upgrades) if you need to upgrade with zero downtime.
+
+To upgrade MarkLogic Docker from release 10.0-1 to the latest release, perform following steps:
```

Comment:
Sorry I didn't comment on this before but I thought it was just a placeholder and would be changed. You should remove the specific version here unless there are directions that are specific to upgrading to from one version to another. Since we don't have any incompatibilities (and we shouldn't ever) between 10.0-x and future 10.x releases, we should indicate as much.

Maybe it should read something like:
"To upgrade a container from marklogic-server:10.x to a later release, perform following steps:

Reply:
Thanks James.
Updated the example version in the readme